packages/localizations/src/chunks/other.cs.ts

```diff
@@ -236,7 +236,7 @@ const otherCs: typeof en =
       "filterOffers": "Filtrovat nabídky",
       "numberOfCommon": "{{number}} společné",
       "offerNotFound": "Nabídka nebyla nalezena. Možná byla smazána autorem",
-      "inputPlaceholder": "Tady napiš zprávu...",
```

Comment:
In this case we would need this translation: `"inputPlaceholder": "Tady napiš zprávu..."`,
I've changed it few sprints ago

Reply:
If you want to update existing translations, you always need to do that in the table otherwise it will be overridden in the next sync. I have updated the table